test/poll.go

```diff
@@ -6,21 +6,36 @@ import (
 	"time"
 )
 
-// Poll repeatedly calls a function until the function returns the correct response or until poll timeout.
-func Poll(t testing.TB, d time.Duration, want interface{}, have func() interface{}) {
+// Eventually repeatedly calls the fn until it returns true or until the duration is exceeded.
+// The fn will be called up to 100 times.
+func Eventually(t testing.TB, duration time.Duration, fn func() bool) {
```

Comment:
[nit] I don't fully understand this. Don't you get the same with `require.Eventually()`?

Reply:
The behavior is the same, the only difference is `require.Eventually` requires a `tick` time argument whereas `test.Poll` just uses `duration / 100`. I thought it would be nice to use something more like `Eventually`, but keeping the tick time. Not a big deal either way.